src/components/Register/Register.test.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import mockRouterOptions from '../Mock';
+import Auth from '../Auth/Auth';
+
+import Register from './Register';
+
+const initialState = {
+  auth: {
+    response: {
+      user: {
+        token: 'rueueuue',
+      },
+    },
+    isLoading: false,
+    errors: {},
+  },
+};
+
+const mockStore = configureMockStore([thunk]);
+const store = mockStore(initialState);
+
+const props = {
+  errors: {},
+  onClose: jest.fn(),
+  signInUser: jest.fn(),
+  registerUser: Promise.resolve({}),
+};
+const wrapper = mount(
+  <Provider store={store}>
+    <Router>
+      <Register {...props} />
+    </Router>
+  </Provider>,
+  mockRouterOptions
+);
+
+describe('<Register /> rendering', () => {
+  it('should render one <Auth>', () => {
+    expect(wrapper.find(Auth)).toHaveLength(1);
+  });
+
+  it('should redirect on login success>', () => {
```

Comment:
The description should say `register` instead of `login`

Reply:
noted